portal/guides/on-demand-minting.mdx

```diff
@@ -25,37 +27,22 @@ Before we start make sure you have the dependencies installed and understand how
 
 After instantiating the `sdk`, we're going to:
 
-1. Build a component for the `connect wallet` button to... connect our wallet and use it to create a signature
+1. Build a component for the `connect wallet` button to connect our wallet and use it to create a signature
 2. Create a unique signature that the owner of the NFT can sell or gift to someone
 3. Allow anyone, holding that unique signature, to claim a NFT
```

Comment:
was weird to me reading this - I see these 3 steps we're gonna build and then it goes straight to 'setup the navigation' and 'useProtectedPage` which doesn't match to any of these 3 points?

Reply:
adjusted the setup